static/src/javascripts/projects/common/modules/discussion/user-avatars.js

```diff
@@ -1,8 +1,8 @@
 define([
     'common/utils/$',
     'bonzo',
-    'common/modules/discussion/api'
-], function ($, bonzo, discussionApi) {
+    'common/modules/avatar/api'
```

Comment:
I think it deserves a separate folder as it is potentially useful for Discussion as well as Identity code/features.

Re: avatar versus api, no strong preference, was just following what other modules do here - for example discussion/api.js.